stsci/imagestats/__init__.py

```diff
@@ -1,17 +1,16 @@
 """
 Compute desired statistics values for input array objects.
-
-:Author: Warren Hack, Christopher Hanley
-
-:License: :doc:`../LICENSE`
-
 """
 import time
+
 import numpy as np
+
 from .histogram1d import histogram1d
 from .computeMean import computeMean
 from ._version import version as __version__  # noqa F401
 
+__all__ = ["histogram1d", "computeMean", "__version__", "ImageStats"]
+
```

Comment:
I do not see what does this adds. You can already import all these things. I do wish modules were not named as the only functions/classes in those modules. Also, why `__version__`?

Reply:
CC: @pllim